test/e2e/performanceprofile/functests/10_performance_ppc/ppc.go

```diff
@@ -193,5 +193,65 @@ var _ = Describe("[rfe_id: 38968] PerformanceProfile setup helper and platform a
 			Eventually(session).Should(gexec.Exit(1))
 		})
 
+		It("[test_id: 54187] PPC generates profile with PerPodPowerManagement workload hint", func() {
+			defaultArgs := []string{
```

Comment:
We can add the default args in a higher layer and use it for all the tests

Reply:
fixed in the latest commit